experiments/run_time.md

```diff
@@ -0,0 +1,19 @@
+**Motivation** 
+
+Pull request #5 (Fig.2) indicated slow convergence. 
```

Comment:
I am not sure if it is a good practice to point the code to pull requests. 
Github is a layer on top of the code, so the PR can point to the code. 
In my opinion, the code should also be readable without looking at GitHub.